failpipeline-sink-plugins.iml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be checked in. Please add a .gitignore to this repo. a good one can be found here: https://github.com/hydrator/example-transform/blob/master/.gitignore